packages/expo-modules-core/src/createWebModule.ts

```diff
@@ -0,0 +1,3 @@
+export function createWebModule<ModuleType = any>(moduleImplementation: ModuleType): ModuleType {
```

Comment:
Can you please add a short note why it's no-op and that the proper implementation of this function exists in `.web.ts` file. I found it confusing to work with such files as you don't immediately see that there is an alternative version and you're looking at the wrong one.